unlock-app/src/__tests__/services/walletService.test.js

```diff
@@ -61,9 +61,56 @@ describe('WalletService', () => {
   beforeEach(() => {
     nock.cleanAll()
     providers = configure().providers
-    walletService = new WalletService(providers)
+    walletService = new WalletService(providers, () => {})
```

Comment:
I see you pass the second argument just to "mock" the setTimeout, right?
I really think this is not the right approach :/